blog-content/README.md

```diff
@@ -0,0 +1,5 @@
+- names of files can only have letters and dashes
+
+TODO:
+- featured shouldn't be required
+- slug shouldn't be required
```

Comment:
is this still accurate?

Reply:
Featured defaults to false, while the slug is autogenerated based on the filename. Updated the sitemap to contain the new blog-content paths.